backend/level4/README.md

```diff
@@ -1,7 +1,10 @@
 # Level 4
 
-In case of an accident we apply a €800 [deductible](http://en.wikipedia.org/wiki/Deductible).
-We want to offer the driver the option to reduce the deductible amount from €800 to €150, for a few more euros per day.
+We want to know how much money must be debited/credited for each actor.
 
-The rule is simple: the driver is charged an additional 4€/day when she chooses the "deductible reduction" option.
-**The money goes to us, not to the owner.**
+Reminder:
+
+- The *owner* receives the rental price minus the commission
+- The *insurance* receives its part of the commission
+- The *assistance* receives its part of the commission
+- *Drivy* receives its part of the commission
```

Comment:
You could make this clearer with 'The insurance provider receives their part of the commission' and 'The assistance provider receives their part of the commission' (and so same [here](https://github.com/drivy/jobs/pull/6/files#diff-f05e4bcf49ea623697df738d73ebfb91R7)

Reply:
Actually reading it again, I'm not sure it's even worth listing this.

How about just "We want to know how much money must be debited/credited for each actor." and that's it?

Reply:
👍